src/data/Register.jsx

```diff
@@ -0,0 +1,10 @@
+export const Registration = {
```

Comment:
make sure to import it and use it as well

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/122486013/562fb6eb-1571-4102-94fc-8cff1f81b782)
I imported it into form/register and used it in the second line

Reply:
make sure to push these changes as well, doesn't look like they are pushed